nservicebus/messaging/routing.md

```diff
@@ -131,30 +131,29 @@ snippet:Routing-CustomRoutingStore
 NOTE: The function passed to the `AddDynamic` call is executed **each time** a message is sent so it is essential to make sure it is performant (e.g. by caching the results if getting the result requires crossing a process boundary).
 
 
-### Endpoint mapping layer
-
-Mapping of an endpoint to the collection of instances is the responsibility of **operations engineers**. They usually know where a given endpoint has been deployed.
-
+### Endpoint instance mapping layer
 
-#### Default mapping
-
-By default, NServiceBus assumes that each endpoint has a single non-scaled-out instance without a *discriminator*. This default allows for running without any configuration on transports like SQL Server or RabbitMQ.
+The purpose of the endpoint instance mapping layer is to provide information about physical deployments of a given endpoint. It is usually the responsibility of **operations engineers** who control environments and have the knowledge where a given endpoint has been deployed. Endpoint instance mapping is optional for most transports in most cases because of built-in conventions. An example of situation where it is required is a scaled out endpoint using MSMQ transport. Other endpoints who send messages to it need to be aware of its instances.
 
 
 #### Using config file
 
-In most scenarios endpoint mapping should be done via a config file so it can be modified without the need for re-deploying the binaries. 
+Usually endpoint mapping should be done via a config file so it can be modified without the need for re-deploying the binaries. 
 
 snippet:Routing-FileBased-Config
 
 To read more see [file-based endpoint mapping](/nservicebus/messaging/file-based-routing.md).
 
+When using transports other than MSMQ the filed-based mapping can be configured via *advanced* API:
+
+snippet:Routing-FileBased-ConfigAdvanced
+
 NOTE: If using a static type mapping to an address instead of an endpoint the advantages of file-based instance resolution will not be possible.
 
 
 #### Static mapping
 
-The `EndpointInstances` class provides a method that allows the registration of a static mapping. This API is useful in very specific scenarios where developers need to take control over a certain mapping.
+The `EndpointInstances` class provides a method that allows the registration of a static mapping. This API is useful in very rare and specific scenarios where developers need to take control over a certain mapping.
```

Comment:
Very rare is kind of rendundant. If the scenario is adequately described then the rareness should be able to be inferred, or not, based on a given environment